src/polyfem/io/OutData.cpp

```diff
@@ -956,6 +957,9 @@ namespace polyfem::io
 
 		reorder_output = args["output"]["data"]["advanced"]["reorder_nodes"];
 
+		// TODO
+		use_hdf5 = false;
```

Comment:
Finish this before merging?

Reply:
Yes, it requires changing the spec, will do once I have time